Polyhedron/demo/Polyhedron/Scene_points_with_normal_item.cpp

```diff
@@ -68,12 +76,17 @@ struct Scene_points_with_normal_item_priv
     }
     normal_Slider = new QSlider(Qt::Horizontal);
     normal_Slider->setValue(20);
+    point_Slider = new QSlider(Qt::Horizontal);
+    point_Slider->setValue(5);
+    point_Slider->setMinimum(1);
+    point_Slider->setMaximum(25);
```

Comment:
Once C++11 will be accepted for the code of CGAL demos, the use of [delegating constructors] will avoid the code duplication when there are several constructors.

http://stackoverflow.com/q/26199431